sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryItemTests.cs

```diff
@@ -119,15 +122,14 @@ public void ValidateTelemetryItem_Activity_WithParentSpanId()
         [InlineData(null)]
         public void HttpMethodAndHttpRouteIsUsedForHttpRequestOperationName(string? route)
         {
-            using ActivitySource activitySource = new ActivitySource(ActivitySourceName);
+            using ActivitySource activitySource = new ActivitySource(RequestActivitySourceName);
             using var activity = activitySource.StartActivity(
-                ActivityName,
+                RequestActivityName,
                 ActivityKind.Server,
                 null,
                 startTime: DateTime.UtcNow);
 
             Assert.NotNull(activity);
-            activity.DisplayName = "/getaction";
```

Comment:
This test was enforcing that the developer set DisplayName is overwritten by the method + route. If this line is removed then the test is now enforcing that the exporter sets a default name.

Still, this "default DisplayName" logic is in the AspNetCore instrumentation, so should not also be in the exporter.